### PR TITLE
Add command eav:attribute:create-dummy-values

### PR DESCRIPTION
**Pull request by giuseppemorelli**
Hi guys,
this version of the command need to be tested on magento version < 1.6.2.0.
Your system has a beautiful test environment with travis ci, so I create this PR to test all the version supported by n98-magerun.

Changes proposed in this pull request:
- add a command to create attribute values (only for dropdown attribute)

### CODE REVIEW

**Comment by giuseppemorelli** [comment]
@ktomk: I personally tested the command in a local environment from 1.6.2.0 to 1.9.2.4 and in each version I didn't find errors (no automated)

In the code there aren't "die" or "exit" that broke your test system (as you find in the last PR).
All the test are "green", but I will continue to test it in a local environment from 1.6.2.0 to your last version supported (1.3?).
I want to be sure that there aren't error with some magento version.

Did you find some errors in the last PR, or I missing something?
Thank you!

File: src/N98/Magento/Command/Eav/Attribute/Create/DummyCommand.php [hunk 1]
```diff
@@ -0,0 +1,410 @@
...
+    {
+        $this->detectMagento($output, true);
+        if(!$this->initMagento()) {
+            return;
+        }
+
+        $output->writeln("<warning>This only create sample attribute values, do not use on production environment</warning>");
+
+        // Ask for Arguments
+        $_argument = $this->askForArguments($input, $output);
```
Comment (tkn98) [inline src/N98/Magento/Command/Eav/Attribute/Create/DummyCommand.php]: Please do not use any underscores at the beginning of a variable name.

**Comment by tkn98** [comment]
Thanks for creating a new PR.

Can you please remove "(new version)" from the subject line of the commit message? And can you rebase on top of the latest develop version?

**Comment by giuseppemorelli** [comment]
@ktomk: with the last push I rebased the commit, change the name and changes variables without underscore at the beginning

**Comment by tkn98** [comment]
@giuseppemorelli: Thanks for the quick response. What do you think about extracting the part of the class that is providing the (random) data and use it as a dependency? So the data-holding and creation is separated from the command itself. You can put the new file just next to the command for a start.

**Comment by giuseppemorelli** [comment]
@tkn98 ok, no problem. I'll send you the commit when done.

**Comment by giuseppemorelli** [comment]
@tkn98: (after travis-ci check) please look at the commit, so you can check if the changes are good with your guidelines. 

Thanks!

Ps. I rebase the commit with the last develop version.

**Comment by tkn98** [comment]
@giuseppemorelli: Please see the log here: https://travis-ci.org/netz98/n98-magerun/jobs/159545411 - it shows problematic parts with the code. You can run the same command on your local box as well w/o the `--dry-run` switch and it should fix most if not all of the highlighted issues.

**Comment by giuseppemorelli** [comment]
@tkn98: new feature to check the "code format"? In the previous PR these errors don't appear. Anyway, fix sent. Waiting for travis test.

**Comment by tkn98** [comment]
@giuseppemorelli: Yes it's new. We had this in Magerun2 for ca. two weeks and I just ported it with the "shared" style into the Magerun 1 Travis build.

This should remove some ambiguities regarding the overall code.

**Comment by tkn98** [comment]
Thanks!

**Pull request merged**